src/babashka/json/internal/jsonista.clj

```diff
@@ -0,0 +1,21 @@
+(ns babashka.json.internal.jsonista
+  (:require [jsonista.core :as j]))
+
+(defn- -key-fn->object-mapper
+  [key-fn]
+  (if-not key-fn
+    j/default-object-mapper
+    (j/object-mapper
+      {:encode-key-fn key-fn
```

Comment:
If one would pass `keyword` as the keyfn, wouldn't it be weird to encode a keyword using `keyword`?

Reply:
It's not actually used, but I can drop it (ie. there's no actual way to pass a `key-fn` into `write-str`.

Reply:
Yes, I think it'd be better to drop it for now, until we explicitly add support for it.